### ISSUE TITLE
Gap between header and body

### ISSUE THREAD

**Issue by ReuelSam** [ReuelSam/ResumeWebsite2024#1]
There's a white gap between header and the body. I'm not sure why. Cannot figure it out.

![image](https://github.com/user-attachments/assets/3029be92-2eea-4438-bb2d-9c13383ee14e)

**Comment by ReuelSam** [comment]
Resolved by PR #2

**Comment by ReuelSam** [comment]
The main problem was in Index.css. the commented lines were the problem makers:

```
body {
  margin: 0;
  display: flex;
  place-items: center;
  /* min-width: 320px; */
  /* min-height: 100vh; */
}
```
